apps/utils/test_credentials.py

```
@@ -0,0 +1,61 @@
+"""
+Test credentials utility for Scopewatch.
+
+This module provides functions to access test credentials from environment variables,
+which are configured as secrets in the GitHub Development environment.
+"""
+
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)